spec/index.bs

```diff
@@ -5562,6 +5603,43 @@ enum GPUStoreOp {
         </div>
 </dl>
 
+<div algorithm>
+    To determine if it's <dfn abstract-op>valid to draw</dfn> with {{GPURenderEncoderBase}} |encoder|
+    run the following steps:
+
+    1. If any of the following conditions are unsatisfied, return `false`:
+        <div class=validusage>
+            - |encoder|.{{GPURenderEncoderBase/[[pipeline]]}} must not be `null`.
+            - Let |pipelineLayout| be |encoder|.{{GPURenderEncoderBase/[[pipeline]]}}.{{GPUPipelineBase/[[layout]]}}.
+            - For each pair of ({{GPUIndex32}} |index|, {{GPUBindGroupLayout}} |bindGroupLayout|) in
+                |pipelineLayout|.{{GPUPipelineLayout/[[bindGroupLayouts]]}}.
+                - Let |bindGroup| be |encoder|.{{GPUProgrammablePassEncoder/[[bind_groups]]}}[|index|].
+                - |bindGroup| must not be `null`.
+                - |bindGroup|.{{GPUBindGroup/[[layout]]}} must be [=group-equivalent=] with |bindGroupLayout|.
+
+            - Let |pipelineDescriptor| be |encoder|.{{GPURenderEncoderBase/[[pipeline]]}}.{{GPURenderPipeline/[[descriptor]]}}.
+            - For each {{GPUIndex32}} |slot| `0` to
+                |pipelineDescriptor|.{{GPURenderPipelineDescriptor/vertexState}}.{{GPUVertexStateDescriptor/vertexBuffers}}.length:
+                - |encoder|.{{GPURenderEncoderBase/[[vertex_buffers]]}}[|slot|] must not be `null`.
+        </div>
+    1. Return `true`.
```

Comment:
Can we have an issue to check for the buffer binding sizes when `minBufferBindingSize` isn't set?

Also this part + having a pipeline could be refactored to be common to all programmable passes (compute and render).

Reply:
Added the issue and refactored the bind group validation so that it applies to any type of encoder.